_kubedee

```diff
@@ -0,0 +1,69 @@
+_kubedee() {
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    commands="controller-ip create create-admin-sa create-user-sa delete etcd-env kubectl-env list smoke-test start start-worker up version --help"
```

Comment:
I think the command suggestion for help should be `help`, not the flag `--help`.